tests/wpt/web-platform-tests/html/browsers/history/joint-session-history/joint-session-history-only-fully-active.html

```diff
@@ -20,7 +20,7 @@
         // is the child, with session length 1, so the joint session length change will be 1.
         // If all documents count, then the grandchild is reachable via the session history,
         // and it has session length 1, so the joint session length change will be 2.
-        assert_equals(2, window.history.length - old_history_len);
+        assert_equals(1, window.history.length - old_history_len);
```

Comment:
@cbrewster The comment seems to imply that`1` is a valid assertion, actually the one corresponding to only having active documents in the session history, what do you think?